api/rest/worker_node.go

```diff
@@ -23,12 +23,14 @@ import (
 
 	"github.com/caicloud/cyclone/api"
 	"github.com/caicloud/cyclone/pkg/log"
+	"github.com/caicloud/cyclone/pkg/osutil"
 	"github.com/caicloud/cyclone/store"
 	"github.com/emicklei/go-restful"
 	docker_client "github.com/fsouza/go-dockerclient"
 )
 
-const WORKER_NODE_DOCKER_VERSION = "1.10.1"
+// WORKER_NODE_DOCKER_VERSION defines the required docker version in worker node.
+const WORKER_NODE_DOCKER_VERSION = "WORKER_NODE_DOCKER_VERSION"
```

Comment:
And now the default is 1.10.1, I think it would be OK.

We could add some tips in doc, for example "This is just for test."